src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.scss

```diff
@@ -316,6 +316,14 @@ li {
   list-style: none;
 }
 
+h3 {
+  font-size: 1.5rem;
+  font-family: var(--primary-font);
+  font-weight: bold;
+  line-height: 2rem;
+  letter-spacing: 0.002em;
```

Comment:
remove package.lock file from PR

Reply:
you are doing it wrong ) 
you need to remove it from PR not from project )

Reply:
you can do git reset soft - to revert a few commit changes and  than git add ...ubs-order-details.component.scss  only
https://stackoverflow.com/questions/24568936/what-is-difference-between-git-reset-hard-head1-and-git-reset-soft-head#:~:text=When%20using%20git%20reset%20%2D%2D,commit%20you%20%22removed%22%20before.

Reply:
fixed)